doc/example/example_ordinal.ipynb

```diff
@@ -0,0 +1,1047 @@
+      "text/plain": [
+       "<Figure size 1850x1050 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from pypesto.visualize import waterfall\n",
+    "\n",
+    "waterfall([res_standard, res_reduced, res_reduced_reparameterized])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Gradient-based optimization\n",
+    "To use the optimal scaling method with a gradient-based optimizer, the only change is in the optimizer used:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimizer = optimize.ScipyOptimizer(\n",
+    "    method=\"L-BFGS-B\",\n",
+    "    options={\"disp\": None, \"ftol\": 2.220446049250313e-09, \"gtol\": 1e-5},\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Running optimization with a gradient-based optimizer"
```

Comment:
Why is the notebook duplicated for two different optimizers? Can just show the gradient-based example, then specify in words that other optimizers, including gradient-free, also work fine?

Reply:
Yes, can do that, it was only gradient-free, so wanted to show the other one as well. But makes sense to show only one.